lib/pbench/test/unit/agent/task/test_results_move.py

```diff
@@ -121,6 +124,49 @@ def test_move_results(monkeypatch, caplog, pytestconfig):
         )
 
         runner = CliRunner(mix_stderr=False)
+
+        # Test --no-delete
+        result = runner.invoke(
+            move.main,
+            args=[
+                TestMoveResults.CTRL_SWITCH,
+                TestMoveResults.CTRL_TEXT,
+                TestMoveResults.TOKN_SWITCH,
+                TestMoveResults.TOKN_TEXT,
+                TestMoveResults.DELN_SWITCH,
+            ],
+        )
+        assert (
+            result.exit_code == 0
+        ), f"Expected a successful operation, exit_code == {result.exit_code:d}"
+        assert (
+            result.stdout
+            == "Status: total # of result directories considered 1, successfully copied 1, encountered 0 failures\n"
+        )
+        # This should raise an unexpected exception if it was not created.
+        (pbrun / f"{name}.copied").unlink()
+
+        # Test --delete (default) with .running directory.
+        (pbrun / name / ".running").mkdir()
```

Comment:
Just to confirm:  the `.running` marker is a _directory_ and not a file?

Reply:
Ah; which answers one of my confusions in the prior pass. But why is it a directory? Are we putting something in it, or is that just to make it a tiny bit harder to delete accidentally?

Reply:
Historically `.running` is always a directory.